nin/frontend/app/scripts/controllers/bottom.js

```diff
@@ -72,6 +72,17 @@
         }
       });
 
+      commands.on('multiplyLoopLengthBy', function(amount) {
+        if ($scope.loopEnd === undefined || $scope.loopStart === undefined) {
+          return;
+        }
+
+        var clampedAmount = Math.max(0, amount),
+            loopLength = $scope.loopEnd - $scope.loopStart;
+
+        $scope.loopEnd = $scope.loopStart + (loopLength * clampedAmount);
```

Comment:
Perhaps clamp the new loopLength to at least 1, so we wont get a loop that starts and ends on the same frame?